File: Week3/problemA.cs
/*
Language: csharp
ID: 61489
QLink: https://quera.org/course/assignments/61489/problems
Author: @Amirkhaskar
*/

using System;

class Program
{
    static void Main(string[] args)
    {
        // Input coordinates for the two flies
        string[] input = Console.ReadLine().Split();
        int x1 = int.Parse(input[0]);
        int y1 = int.Parse(input[1]);
        int z1 = int.Parse(input[2]);
        int x2 = int.Parse(input[3]);
        int y2 = int.Parse(input[4]);
        int z2 = int.Parse(input[5]);

        // Check if the flies see each other
        if (x1 == x2 || y1 == y2 || z1 == z2)
        {
            Console.WriteLine("YES");
        }
        else
        {
            Console.WriteLine("NO");
        }
    }
}

File: Week3/problemA.py
# Language: python
# ID: 61489
# QLink: https://quera.org/course/assignments/61489/problems
# Author: @Amirkhaskar

x1, y1, z1 = map(int, input().split())
x2, y2, z2 = map(int, input().split())
if x1 == x2 or y1 == y2 or z1 == z2:
    print("YES")
else:
    print("NO")